content/sql/concepts/procedures/procedures.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Procedures'
+Description: 'Procedures are blocks of SQL code that are saved in a database and can be executed repeatedly on demand.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'MySQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+Procedures are blocks of SQL code that are saved in a database and can be executed repeatedly on demand. Also referred to as stored procedures — they can be defined with parameters to be used within the body of the procedure. They can also define parameters that return an output value.
```

Comment:
This last sentence feels like it should be better incorporated. Perhaps this content should be added to second sentence - can be defined with parameters to be used with the body of the procedure or to return a value.